app/controllers/auth_controller.rb

```diff
@@ -0,0 +1,4 @@
+# Controller for the DAHLIA homepage.
+class AuthController < HomeController
```

Comment:
I don't think this should extend HomeController, instead it should extend `ApplicationController` and define its own `useReactApp` override like

```
  def use_react_app
    ENV['SIGN_IN_PAGE_REACT'].to_s.casecmp('true').zero?
  end
```

Reply:
Also, we want a separate feature flag for this page since the homepage will be ready for launch before this page is.

Reply:
Sure, that make sense. But actually `use_react_app` should be a `protected` method. `public` methods are reserved for actions as of Rails convention. Will make amendments.

Reply:
sounds good!